use-timescale/integrations/apache-airflow.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Integrate Apache Airflow with Timescale Cloud
+excerpt: How to install the psql client for PostgreSQL
+products: [cloud, mst, self_hosted]
+keywords: [connect, integrate, apache, airflow]
+---
+
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
+
+# Integrate Apache Airflow with $CLOUD_LONG
+
+Apache Airflow® is a platform created by the community to programmatically author, schedule and monitor workflows.
+
+A [DAG (Directed Acyclic Graph)][Airflow-DAG] is the core concept of Airflow, collecting [Tasks][Airflow-Task] together,
+organized with dependencies and relationships to say how they should run. You declare a DAG in a Python file
+in the `$AIRFLOW_HOME/dags` folder of your Airflow instance.
+
+This page shows you how to use a Python connector in a DAG to integrate Apache Airflow with a $SERVICE_LONG.
+
+## Prerequisites
+
+<IntegrationPrereqs />
+
+* [Install Python3 and pip3][install-python-pip] 
+* [Install Apache Airflow][install-apache-airflow]
+
+   Ensure that your Airflow instance has network access to $CLOUD_LONG.
+
+This example DAG uses the `company` table you create in  [Create regular PostgreSQL tables for relational data][create-a-table-in-timescale]
```

Comment:
Can we either make this into another bullet in prereqs ("Connect to your service and create a table") or add it as the first step in the exchange data section? It's a bit floating now.

Reply:
Tricky. I don't want the user to think they have to use that procedure to make it work, it is more to say this is what we did. I'd prefer to leave as is.